core/storage/feedback/gae_models.py

```diff
@@ -304,7 +304,26 @@ class UnsentFeedbackEmailModel(base_models.BaseModel):
     # Each element in this list is a dict with keys 'exploration_id',
     # 'thread_id' and 'message_id'; this information is used to retrieve
     # corresponding FeedbackMessageModel instance.
-    feedback_messages = ndb.JsonProperty(repeated=True)
+    feedback_message_references = ndb.JsonProperty(repeated=True)
     # The number of failed attempts that have been made (so far) to
     # send an email to this user.
     retries = ndb.IntegerProperty(default=0, required=True, indexed=True)
+
+    @classmethod
+    def create_or_modify(cls, owner_id, feedback_message_reference):
+        """Creates a new instance of UnsentFeedbackEmailModel for sending
+        feedback message email"""
```

Comment:
Docstring should also include information about the return value and its significance. Don't be afraid to write more.